webapp/components/interop-dashboard.js

```diff
@@ -672,5 +708,74 @@ class InteropDashboard extends PolymerElement {
     this.$.toggleStable.setAttribute('aria-pressed', true);
     this.$.toggleExperimental.setAttribute('aria-pressed', false);
   }
+
+  getSortIcon(index) {
+    if (this.toggleStates[index]) {
+      return '/static/expand_more.svg';
+    }
+    return '/static/expand_less.svg';
+  }
+
+  sortRows(rows, index, tableState) {
+    if(index === 0) {
+      const sortedFeatureOrder = [];
+
+      // Alphabetize by focus area column
+      if(tableState.sortColumn === -1) {
+        const rowNames = [];
+        for(let i = 0; i < rows.length; i++) {
+          const feature = rows[i];
+          rowNames[i] = [feature, this.getRowInfo(feature, 'description')];
+        }
+        rowNames.sort((a, b) => {
+          a[1] - b[1];
+        });
+
+        for (let i = 0; i < rowNames.length; i++) {
+          sortedFeatureOrder[i] = rowNames[i][0];
+        }
+        // List in ascending order by score
+      } else if (tableState.sortColumn >= 0) {
+        const individualScores = [];
+
+        for (let i = 0; i < rows.length; i++) {
+          const feature = rows[i];
+          individualScores[i] = [feature, parseFloat(this.getBrowserScoreForFeature(tableState.sortColumn, feature))];
+        }
+        individualScores.sort((a, b) => a[1] - b[1]);
+
+        for (let i = 0; i < individualScores.length; i++) {
+          sortedFeatureOrder[i] = individualScores[i][0];
+        }
+      }
+      if (tableState.sortOrder === 'dec') {
+        sortedFeatureOrder.reverse();
+      }
+      return sortedFeatureOrder;
+    } else {
+      return rows;
+    }
+  }
+
+  handleSortClick(e) {
+    const i = parseInt(e.target.id.split('-')[1]) - 1;
+
+    if (this.tableState.sortColumn !== i) {
+      this.tableState.sortColumn = i;
+      this.tableState.sortOrder = 'asc';
+      this.toggleStates.fill(false);
+      this.toggleStates[i + 1] = true;
+    } else if (this.tableState.sortColumn === i && this.tableState.sortOrder === 'asc') {
+      this.tableState.sortOrder = 'dec';
+      this.toggleStates[i + 1] = false;
+    } else if (this.tableState.sortColumn === i && this.tableState.sortOrder === 'dec') {
+      this.tableState.sortOrder = 'asc';
+      this.toggleStates.fill(false);
+      this.toggleStates[i + 1] = true;
+    }
+    this.tableState.sortColumn = i;
+
+    this.sortToggled = !this.sortToggled;
```

Comment:
I think the reason it felt like the `sortToggled` variable is needed is because the `sortColumn` variable lives as a property in the `tableState` object. Polymer doesn't explicitly know when you've changed values on object properties, so it will not trigger a re-render unless the instance variables explicitly change. If/when `sortColumn` is moved to an actual instance variable rather than inside the `tableState` object, Polymer should know that it needs to re-render anything that uses `sortColumn`.